src/gluonnlp/data/transforms.py

```diff
@@ -704,26 +718,26 @@ class BasicTokenizer():
     performs invalid character removal (e.g. control chars) and whitespace.
     tokenize CJK chars.
```

Comment:
Related to previous PR, should this really be called 'BasicTokenizer? As far as in understand, splitting CJK is useful for BERT models. But BasicTokenizer somewhat implies this should be used / useful everywhere. What do you think @eric-haibin-lin With respect to English models, replacing a single unknown (Chinese) word, with X unknown (Chinese) characters may have an adverse effect?

Reply:
Reasonable hints, maybe ` BERTBasicTokenizer ` is more appropriate.  How is it? @eric-haibin-lin @leezu